base/compiler/abstractinterpretation.jl

```diff
@@ -1837,19 +1837,18 @@ function abstract_call_opaque_closure(interp::AbstractInterpreter,
     tt = closure.typ
     sigT = (unwrap_unionall(tt)::DataType).parameters[1]
     match = MethodMatch(sig, Core.svec(), closure.source, sig <: rewrap_unionall(sigT, tt))
+    ⊑ₚ = ⊑(ipo_lattice(interp))
     const_result = nothing
     if !result.edgecycle
         const_call_result = abstract_call_method_with_const_args(interp, result,
             nothing, arginfo, si, match, sv)
         if const_call_result !== nothing
-            if const_call_result.rt ⊑ rt
+            if const_call_result.rt ⊑ₚ rt
```

Comment:
It turns out that the `ManifoldsBase` test suite needs this change also.

Reply:
Ah, I mean, the corresponding change in `abstract_invoke`. These changes in `abstract_call_opaque_closure` are driven by that.